docs/01-overview/main-areas/observability/obsv-03-istio-monitoring.md

```diff
@@ -1,22 +1,28 @@
 
-The Istio-related instance is a Deployment named `monitoring-prometheus-istio-server`, configured with a short data retention time and hardcoded configuration that you should not change. It also has no PersistentVolume attached. This instance never discovers additional metric endpoints from such resources as ServiceMonitors.
+2. A dedicated Prometheus instance scrapes and aggregates the service-level metrics. That instance is configured with the smallest possible data retention time because the raw metrics scraped from the Istio Proxies have high-cardinality and don't need to be kept further. Instead, the main Prometheus instance scrapes the aggregated metrics through the `/federate` endpoint. This instance is a Deployment named `monitoring-prometheus-istio-server`, with a hardcoded configuration you should not change. It also has no PersistentVolume attached. This instance never discovers additional metric endpoints from such resources as ServiceMonitors.
 
-The monitoring chart is configured in such a way that it is possible to scrape metrics using [`Strict mTLS`]
-(https://istio.io/latest/docs/tasks/security/authentication/authn-policy/#globally-enabling-istio-mutual-tls-in-strict-mode). For this to work, Prometheus is configured to scrape metrics using Istio certificates. Prometheus is deployed with a sidecar proxy which rotates SDS certificates and outputs them to a volume mounted to the 
-corresponding Prometheus container. To stick to Istio's observability best practices, Prometheus's Istio-proxy is configured to not intercept or redirect any traffic. By default, metrics from Kyma components are scraped using mTLS with some exceptions (i.e. Kiali, Prometheus Operator, Prometheus Istio) due to several reasons.
+3. The main Prometheus instance is configured to support scraping metrics using [`Strict mTLS`](https://istio.io/latest/docs/tasks/security/authentication/authn-policy/#globally-enabling-istio-mutual-tls-in-strict-mode). For this to work, Prometheus is configured to scrape metrics using Istio certificates. 
+   
+4. Prometheus is deployed with a sidecar proxy which rotates SDS certificates and outputs them to a volume mounted to the corresponding Prometheus container. It is configured to not intercept or redirect any traffic. 
+   
+5. By default, metrics from Kyma components are scraped using mTLS. The components' workload is part of the Istio infrastructure, they are deployed with a sidecar proxy, and scraped using mTLS.  As an exception, components deployed without sidecar proxy are scraped using http (for example, controllers like Prometheus operator).
```

Comment:
"The components' workload is part of the Istio infrastructure" - The workload is not part of the istio infrastructure in my pov.